internal/service/pushbasedlogexport/resource_schema.go

```diff
@@ -0,0 +1,96 @@
+				MarkdownDescription: "Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.\n\n**NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+				Validators: []validator.String{
+					stringvalidator.LengthBetween(24, 24),
+					stringvalidator.RegexMatches(regexp.MustCompile("^([a-f0-9]{24})$"), ""),
+				},
+			},
+			"iam_role_id": schema.StringAttribute{
+				Optional:            true,
+				Description:         "ID of the AWS IAM role that will be used to write to the S3 bucket.",
+				MarkdownDescription: "ID of the AWS IAM role that will be used to write to the S3 bucket.",
+			},
+			"links": schema.ListNestedAttribute{
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"href": schema.StringAttribute{
+							Computed:            true,
+							Description:         "Uniform Resource Locator (URL) that points another API resource to which this response has some relationship. This URL often begins with `https://cloud.mongodb.com/api/atlas`.",
+							MarkdownDescription: "Uniform Resource Locator (URL) that points another API resource to which this response has some relationship. This URL often begins with `https://cloud.mongodb.com/api/atlas`.",
+						},
+						"rel": schema.StringAttribute{
+							Computed:            true,
+							Description:         "Uniform Resource Locator (URL) that defines the semantic relationship between this resource and another API resource. This URL often begins with `https://cloud.mongodb.com/api/atlas`.",
+							MarkdownDescription: "Uniform Resource Locator (URL) that defines the semantic relationship between this resource and another API resource. This URL often begins with `https://cloud.mongodb.com/api/atlas`.",
+						},
+					},
+				},
+				Computed:            true,
+				Description:         "List of one or more Uniform Resource Locators (URLs) that point to API sub-resources, related API resources, or both. RFC 5988 outlines these relationships.",
+				MarkdownDescription: "List of one or more Uniform Resource Locators (URLs) that point to API sub-resources, related API resources, or both. RFC 5988 outlines these relationships.",
+			},
+			"prefix_path": schema.StringAttribute{
+				Optional:            true,
+				Computed:            true,
+				Default:             stringdefault.StaticString(""),
```

Comment:
So in TPF if attribute is marked as default it needs to be set to Computed too otherwise TF throws error. 
As for `""` this is a requirement from the API

Reply:
👍 the default "" coming for the API does seem a little odd

Reply:
right, but they are adding a mention about this in the API spec and currently this is how the UI is set up too. The idea is "" means the root directory of S3 bucket and it is a required attribute by the API so I think it makes sense.